app/src/main/java/ch/blinkenlights/android/vanilla/ThemeHelper.java

```diff
@@ -33,6 +51,124 @@ final public static void setTheme(Context context, int theme) {
 		context.setTheme(getThemeResource(context, theme));
 	}
 
+	/**
+	 *
+	 */
+	final private static String getColor(){
+		return "#b13fb5";
+	}
+
+	/**
+	 *
+	 */
+	final private static int getParsedColor(){
+		return Color.parseColor(getColor());
+	}
+
+	/**
+	 *
+	 */
+	final private static int getParedDarkColor(){
+		return ColorUtils.blendARGB(getParsedColor(), Color.BLACK, 0.15f);
+	}
+
+
+	/**
+	 * Sets the accentcolor to the value stored in the preferences.
+	 */
+	final public static void setAccentColor(Activity a) {
+
+
+		int myColor = getParsedColor();
+		int myColorDark = getParedDarkColor();
+
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+			a.getWindow().setStatusBarColor(myColorDark);
+		}
+
+		ActionBar actionBar = a.getActionBar();
+		if(actionBar!=null){
+			ColorDrawable colorDrawable = new ColorDrawable(myColor);
+			actionBar.setBackgroundDrawable(colorDrawable);
+		}
+
+		SeekBar mSeekBar = (SeekBar)a.findViewById(R.id.seek_bar);
```

Comment:
So this code just looks out for 'known' UI elements?

I don't think that's a good approach: UI elements have styles, their name (or ID) shouldn't just magically give them a color.

I'd rather expect this function to walk trough all UI nodes (starting at the root window) and search for a custom attribute to decide whether or not the color needs to change.

Reply:
Sadly the styles do not allow me to override their colors, so i needed to do it directly on the element itself.

How would i do that, i have tried to find specific elements for the settings and the Jump to Time Dialog, but that did not work at all. Either i cannot access the proper view or it just does not return the element i want (or mostly, none at all)